package.json

```diff
@@ -73,12 +73,14 @@
     "@shoelace-style/shoelace": "^2.15.1",
     "@types/jest": "^29.5.12",
     "@types/node": "^20.13.0",
+    "@web/dev-server-esbuild": "^1.0.2",
```

Comment:
Are these two rebuild dependencies needed?  npm start and webtest seem to work without them.